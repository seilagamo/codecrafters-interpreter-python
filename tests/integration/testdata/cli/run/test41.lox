var bar = "global bar";
var world = "global world";
var hello = "global hello";
{
  var bar = "outer bar";
  var world = "outer world";
  {
    var bar = "inner bar";
    print bar;
    print world;
    print hello;
  }
  print bar;
  print world;
  print hello;
}
print bar;
print world;
print hello;