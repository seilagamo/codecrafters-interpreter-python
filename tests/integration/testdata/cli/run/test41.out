inner bar
outer world
global hello
outer bar
outer world
global hello
global bar
global world
global hello
